tensorbay/opendataset/AnimalPose/loader.py

```diff
@@ -153,15 +152,14 @@ def _get_data_part2(root_path: str, aniamls: Iterable[str]) -> Iterator[Data]:
                 labels = xmltodict.parse(fp.read())
 
             box2d = labels["annotation"]["visible_bounds"]
-            data.label.box2d = [
-                LabeledBox2D(
-                    x=float(box2d["@xmin"]),
-                    y=float(box2d["@xmax"]),  # xmax means ymin in the annotation
-                    width=float(box2d["@width"]),
-                    height=float(box2d["@height"]),
-                    category=animal,
-                )
-            ]
+            labeled_box2d = LabeledBox2D.from_xywh(
```

Comment:
We also need `LabeledBox2D.from_xywh()` and `LabledBox2D.from_center_xywh()` which act like `LabeledBox2D.__init__` which can set category and attributes